### ISSUE TITLE
feat[dev]: ✨ implement A11y improvements for Quote Template

### ISSUE THREAD

**Issue by yoezlem** [solid-design-system/solid#1511]
## Description
This task describes the Storybook implementation for the Quote template as part of the A11y improvements outlined in Epic https://github.com/solid-design-system/solid/issues/1465.

A11y improvements for the Quote template can be found here: [Quote Notes](https://www.notion.so/matuzo/Quote-113f3f6e1bc080e0bdc9d251e0dc929c).

------------------------------------

## Comment
No critical issues found. 

## Notes
- [x] For block quotes use the appropriate element, `<blockquote>`. `<q>` is only for short inline quotes => as we don't offer an inline-quote in the template, all of our examples should be a blockquote => add a11y hint.
- [ ] Please use an image here that fits the purpose. The name is "Jane Miller," but the description says, "A caring father, balancing one child on his shoulder and gently cradling another in his arms.". In a quote, you'd likely rather see a portrait. If you use a portrait, you must decide whether you want to describe it or not. If not, don't repeat the person's name; leave the alt-attribute empty => switch and zoom in on a placeholder image from our library (see [figma](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=4131-17720&t=QnODZ2LBFSrnK4G4-4) implementation).
  - [ ] create a new **_square_** "profile" image in the placeholder package with the current image ("coffeshop") from figma. This can later be replaced by an official "profile" image, aligned with brand.
  - [x] change quote author to "Jane"
  - [ ] change description in code accordingly
  - [x] change alt text in code accordingly

## Best Practices
- If you use an image, don't repeat the person's name in the alt-attribute. Either describe how the person looks like or leave the description empty.

## DoR
- [x] Item has been estimated by the team
- [x] Item dependencies have been identified and documented

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by MartaPintoTeixeira** [comment]
Note: Read figma comments regarding quote image.

**Comment by MartaPintoTeixeira** [comment]
Figma template samples have been updated to provide guidance for dev.
